gradle/gooeycli/src/main/groovy/org/terasology/cli/commands/InitCommand.groovy

```diff
@@ -30,14 +30,15 @@ class InitCommand extends BaseCommandType implements Runnable {
 
     void run() {
         println Ansi.AUTO.string("@|bold,green,underline Time to initialize $distro !|@")
-        def targetDistroURL = "https://raw.githubusercontent.com/$githubOrg/Index/master/distros/$distro/gradle.properties"
+        String origin = gitOptions.resolveOrigin()
```

Comment:
How does this resolve? We'd run this command in the `MovingBlocks/Terasology` repository, but the index repo is located in a different org, `Terasology/Index`...

Reply:
because it's resolving to Terasology?